splash/har/qt.py

```diff
@@ -114,13 +115,17 @@ def reply2har(reply, include_content=False, binary_content=False):
 
     status_text = reply.attribute(QNetworkRequest.HttpReasonPhraseAttribute)
     if status_text is not None:
-        res["statusText"] = bytes(status_text).decode('latin1')
+        try:
+            res["statusText"] = bytes(status_text, 'latin1').decode('latin1')
+        except TypeError:
+            res["statusText"] = bytes(status_text).decode('latin1')
```

Comment:
status_text is not unicode. It's a Qt byte array. So it needs to be converted into bytes and then decoded into unicode.